### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by DefCon-007**
I have added the modal someone work on the frontend part.
Ping @kossiitkgp/maintainers 

screen shot 2016-11-19 at 11 26 18 am

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -433,300 +441,328 @@
...
+
+                    <div class="field-wrap">
+                        <input type="text" name="lname" id="lname" placeholder="Last Name" autocomplete="off" required>
+                    </div>
+                </div>
 
-              <div class="top-row">
                 <div class="field-wrap">
-                  <input type="text" name="fname" id="fname" placeholder="First Name" autocomplete="off" required>
+                    <input type="rollno" name="rollno" id="rollno" placeholder="Roll number" autocomplete="off" required>
```
Comment (OrkoHunter) [inline templates/index.html]: What is `type="rollno"` ? Isn't the type of input text here?

**Comment by OrkoHunter** [comment]
@DefCon-007 It seems that you have re-added the removed js and css in the index file. :/

**Comment by OrkoHunter** [comment]
No wait, they have not been removed yet. #27

**Comment by nitinkgp23** [comment]
We are done with this, i suppose.

**Pull request closed without merging**